.github/workflows/LHCI-upload-to-private-server.yml

```
@@ -11,5 +11,6 @@ jobs:
         with:
           urls: |
             https://treo.sh/
-          upload.serverBaseUrl: ${{ secrets.LHCI_SERVER }}
-          upload.token: ${{ secrets.LHCI_TOKEN }}
+          # FIXME:
```

Is this a TODO for us, or for the user if they want to use it?

```suggestion
          # TODO(action-user): Set lhci-server secrets here.
```

---

that's for me, for some reason Treo's private LHCI server returned 502, and I need to debug it 🔍